az-digital/az_quickstart
# Fixes #515 Version indicator changes

This PR updates the version indicator on the Quickstart toolbar.

## Related issues
#515

## How to test

- Test locally
- Add version number to `az_quickstart.info.yml`
- Clear cache
- Verify version indicator appears

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
